db/migrate/20151016231742_update_sample_chargeback_rate.rb

```diff
@@ -0,0 +1,9 @@
+class UpdateSampleChargebackRate < ActiveRecord::Migration
+  def up
+    ChargebackRate.where(:default => true).update_all(:description => "Sample")
+  end
+
+  def down
+    ChargebackRate.where(:default => true).update_all(:description => "Default")
```

Comment:
This looks wrong.  You probably meant:
`ChargebackRate.where(:description => "Default").update_all(:default => true)`